### ISSUE TITLE
error on smartphone

### ISSUE THREAD

**Issue by dotmobo** [woofers/react-pico-8#76]
Hi,

On desktop it works fine !

But on smartphone, i have the following issue in the console :

react-pico-8.dev.js:1 Uncaught ReferenceError: left is not defined
react-pico-8.dev.js:1 Uncaught ReferenceError: right is not defined

How can i fix it ?

Thanks

**Comment by dotmobo** [comment]
It worked on 3.3.1 version. 
It's a bug since the 4.0.0 version.

**Comment by woofers** [comment]
Hey @dotmobo,

Thanks for opening the issue, this should be fixed in `v4.0.3` by 7e85b0d, looks like I broke something while I was refactoring to CSS modules instead of emotion for v4.

At the moment this library does not really fully support mobile, the game will run but without #3 the only way to control it would be to use a Bluetooth gamepad.  If your interested I did start some work on [this branch](https://github.com/woofers/react-pico-8/tree/mobile) but never got around to finishing it.
